lib/RGBot.js

```diff
@@ -638,6 +749,7 @@ class RGBot extends EventEmitter {
     /**
      * Attempt pathfinding. If the Bot becomes 'stuck' then cancel pathfinding.
      * The Bot is considered 'stuck' if it fails to move or perform mining/crafting/chest-interaction actions during a specified interval.
+     * If calling from python this function will not do stuck handling as the python/js bridge cannot handle timers/intervals.
```

Comment:
This is misleading because it makes it sound like I'll still perform the movement, I just won't have stuck detection but this just immediately returns `!stuck` if I'm python